test/QuestFactory.t.sol

```diff
@@ -247,18 +247,20 @@ contract TestQuestFactory is Test, Errors, Events, TestUtils {
     }
 
     function test_claimCompressed_erc20_mocked_data() public{
-        address participantMocked = 0xde967dd32C1d057B368ea9F37d70469Cd7F6bF38;
-        address referrerMocked = address(0);
-        bytes32 txHash = 0x57498a77018f78c02a0e2f0d0e4a8aab048b6e249ff936d230b7db7ca48782e1;
         uint32 txHashChainId = 1;
-        bytes16 questId = 0x88e08cb195e64832845fa92ec8f2034a;
-        string memory questIdString = "88e08cb1-95e6-4832-845f-a92ec8f2034a";
-        string memory actionType = "other";
-        string memory questName = "Advanced Trading with dYdX";
-        bytes32 r = 0x12a1078fd9cf9bbed1fa8f00b9abd75baa6c07073706479ca25ec34f4043b327;
-        bytes32 vs = 0x5aca8bfe397d45aa673c07d2ce845cb4419eadc0cbe8977de337799a37b2e1a3;
+        bytes32 txHash = hex'57498a77018f78c02a0e2f0d0e4a8aab048b6e249ff936d230b7db7ca48782e1';
+        bytes16 questId = hex'88e08cb195e64832845fa92ec8f2034a';
```

Comment:
Is there any benefit for using `hex'...'`? Or is it just preference?

Reply:
I'm under the impression just preference for readability and a compilation error if we copy/paste incorrectly.

I was trying to be consistent with the existing code and saw the hex literal used pretty commonly